spec/spec_helper_acceptance.rb

```diff
@@ -1,38 +1,20 @@
-ENV['PUPPET_INSTALL_TYPE'] ||= 'agent'
-ENV['BEAKER_IS_PE'] ||= 'no'
-ENV['BEAKER_PUPPET_COLLECTION'] ||= 'puppet6'
-ENV['BEAKER_debug'] ||= 'true'
+require 'voxpupuli/acceptance/spec_helper_acceptance'
+
 ENV['BEAKER_setfile'] ||= 'centos7-64{hostname=centos7-64.example.com}'
-ENV['BEAKER_HYPERVISOR'] ||= 'docker'
-
-require 'beaker-puppet'
-require 'beaker-rspec'
-require 'beaker/puppet_install_helper'
-require 'beaker/module_install_helper'
-
-run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
-install_module_on(hosts)
-install_module_dependencies_on(hosts)
-install_module_from_forge('camptocamp-systemd', '>= 2.0.0 < 3.0.0')
```

Comment:
https://github.com/theforeman/puppet-puppet/pull/710 will be needed to address this.